tests/TestCase/ORM/EntityTest.php

```diff
@@ -132,6 +132,27 @@ public function testExtractOriginal()
     }
 
     /**
+     * Test that all original values are returned properly
+     *
+     * @return void
+     */
+    public function testExtractOriginalValues()
+    {
+        $entity = new Entity([
+            'id' => 1,
+            'title' => 'original',
+            'body' => 'no',
+            'null' => null,
+        ], ['markNew' => true]);
+        $entity->set('body', 'updated body');
+        $result = $entity->getOriginalValues(['id', 'title', 'body', 'null']);
+        $expected = [
+            'body' => 'no'
```

Comment:
If the value has not been updated, I would expect that the current value is also included, as it is 'original' too.